website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Quickstart for dbt Core using DuckDB
+id: duckdb
+description: "Learn to use dbt Core using DuckDB."
+hoverSnippet: "Learn to use dbt Core using DuckDB."
+platform: 'dbt-core'
+icon: 'duckdb-seeklogo'
+level: 'Beginner'
+hide_table_of_contents: true
+tags: ['dbt Core','Quickstart']
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+In this quickstart guide, you'll learn how to use dbt Core with DuckDB, enabling you to get set up quickly and efficiently. [DuckDB](https://duckdb.org/) is an open-source database management system which is designed for analytical workloads. It is designed to provide fast and easy access to large datasets, making it well-suited for data analytics tasks. 
+
+
+This guide will demonstrate how to: 
+
+- Create a virtual development environment using a template provided by dbt Labs.
+   - This sets up a fully functional dbt environment with an operational and executable project. The codespace automatically connects to the DuckDB database and loads a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
```

Comment:
These instructions appear to remove the Codespace workflows and run everything locally. Is that correct?

```suggestion
   - This creates a fully functional dbt environment with an operational and executable project. The DuckDB database will be installed locally on your machine and load a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
```